modules/Microsoft.Compute/virtualMachines/.test/linux.atmg/dependencies.bicep

```diff
@@ -59,7 +59,7 @@ resource sshDeploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01'
     }
   }
   properties: {
-    azPowerShellVersion: '6.2.1'
+    azPowerShellVersion: '9.1'
```

Comment:
Missing update to msiRGContrRoleAssignment name above as done for others

Reply:
done